doc/concepts/recipe.rst

```diff
@@ -1,22 +1,77 @@
 .. _modelrecipe:
 
 Recipes
-===============
+=======
 
 An Arbor *recipe* is a description of a model. The recipe is queried during the model
 building phase to provide information about cells in the model, such as:
 
-  * the number of cells in the model;
-  * the type of a cell;
-  * a description of a cell, e.g. with soma, synapses, detectors, stimuli;
-  * the number of spike targets;
-  * the number of spike sources;
-  * the number of gap junction sites;
-  * incoming network connections from other cells terminating on a cell;
-  * gap junction connections on a cell.
+  * The **number of cells** in the model.
+  * The **kind** of each cell.
+  * The **description** of each cell, e.g. with morphology, synapses, detectors, stimuli.
+  * The number of **spike targets**.
+  * The number of **spike sources**.
+  * The number of **gap junction sites**.
+  * Incoming **network connections** from other cells terminating on a cell.
+  * **Gap junction connections** on a cell.
+  * **Probes** on a cell.
+
+To better illustrate the content of a recipe, let's consider the following model of
```

Comment:
In case **description** includes all properties of a cable cell, I'd say sth like "To better illustrate the content of a recipe, let's consider a model of three cells. We give each of these three cells a **description**:"

Reply:
It's not all the properties. I'll try to make it clearer below in the example.